src/test/regress/bin/create_test.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+import sys
+import random
+import os
+
+if len(sys.argv) != 2:
+    raise Exception("Expected the name of the new test as an argument")
+
+test_name = sys.argv[1]
+
+filename = f"src/test/regress/sql/{test_name}.sql"
+
+if os.path.isfile(filename):
+    raise Exception(f"test file '{filename}' already exists")
+
+shard_id = random.randint(1, 999999) * 100
+
+contents = f"""CREATE SCHEMA {test_name};
```

Comment:
Can you move this to the top(with shard_id) so that it doesn't intervene with the logic?